wikitemplate-android.md

```diff
@@ -2,105 +2,108 @@
 
 ## Autofill tests
 
-1. [ ] Test that autofill works on http://www.roboform.com/filling-test-all-fields
+- [ ] Test that autofill works on http://www.roboform.com/filling-test-all-fields
 
 ## Zoom
 
-1. [ ] Test zoom in / out gestures work
-2. [ ] Test that navigating to a different origin resets the zoom
+- [ ] Test zoom in / out gestures work
+- [ ] Test that navigating to a different origin resets the zoom
 
 ## Bravery settings
 
-1. [ ] Check that HTTPS Everywhere works by loading https://https-everywhere.badssl.com/
-2. [ ] Turning HTTPS Everywhere off and shields off both disable the redirect to https://https-everywhere.badssl.com/
-3. [ ] Not Yet Implemented - Check that ad replacement works on http://slashdot.org
-4. [ ] Check that toggling to blocking and allow ads works as expected.
-5. [ ] Test that clicking through a cert error in https://badssl.com/ works.
-6. [ ] Test that Safe Browsing works (http://downloadme.org)
-7. [ ] Turning Safe Browsing off and shields off both disable safe browsing for http://downloadme.org/.
-8. [ ] Visit https://brianbondy.com/ and then turn on script blocking, nothing should load. Allow it from the script blocking UI in the URL bar and it should work.
-9. [ ] Test that about:preferences default Bravery settings take effect on pages with no site settings.
-10. [ ] Not Yet Implemented - Test that turning on fingerprinting protection in about:preferences shows 3 fingerprints blocked at https://jsfiddle.net/bkf50r8v/13/. Test that turning it off in the Bravery menu shows 0 fingerprints blocked.
-11. [ ] Test that 3rd party storage results are blank at https://jsfiddle.net/7ke9r14a/7/ when 3rd party cookies are blocked.
-12. [ ] Not Yet Implemented - Test that audio fingerprint is blocked at https://audiofingerprint.openwpm.com/ when fingerprinting protection is on.
+- [ ] Check that HTTPS Everywhere works by loading https://https-everywhere.badssl.com/
+- [ ] Turning HTTPS Everywhere off and shields off both disable the redirect to https://https-everywhere.badssl.com/
+- [ ] Not Yet Implemented - Check that ad replacement works on http://slashdot.org
+- [ ] Check that toggling to blocking and allow ads works as expected.
+- [ ] Test that clicking through a cert error in https://badssl.com/ works.
+- [ ] Test that Safe Browsing works (http://downloadme.org)
+- [ ] Turning Safe Browsing off and shields off both disable safe browsing for http://downloadme.org/.
+- [ ] Visit https://brianbondy.com/ and then turn on script blocking, nothing should load. Allow it from the script blocking UI in the URL bar and it should work.
+- [ ] Test that about:preferences default Bravery settings take effect on pages with no site settings.
+- [ ] Test that 3rd party storage results are blank at https://jsfiddle.net/7ke9r14a/7/ when 3rd party cookies are blocked.
+
+**Fingerprinting Tests (Enable fingerprinting protection in Settings)**
```

Comment:
Should the mobile templates use the same new FP cases as `wikitemplate.md`?

Example: `got canvas fingerprint 0`  and  `got webgl fingerprint 00`

Reply:
@diracdeltas  could confirm if the expected result is same for laptop and mobile